details.md

```diff
@@ -1,15 +1,25 @@
-# Portfolio plans
+# Portfolio Plans
 
-**View portfolio level work items across projects**
+## Portfolio Plans
 
-#### Overview
+**View portfolio level work items across projects.** Portfolio Plans is an organizational tool that helps teams drive cross-project visibility and alignment by tracking portfolio level work in a calendar view.
 
-TODO
+## Hand-pick portfolio items to track
 
-#### How does it work?
+To get started in a new plan, select the items that you wish to track. First, select a project. Then, select the item(s) you wish to track on the plan. You may add items from multiple projects.

```

Comment:
How about this..

To get started is easy. First, create a new plan. Then select a project and add the item(s) you wish to track on the plan. You may add as many work items as you want and from multiple projects.